cmd/kube-proxy/app/server.go

```diff
@@ -470,12 +469,13 @@ func (s *ProxyServer) Run() error {
 	}
 
 	if len(s.ResourceContainer) != 0 {
-		// Run in its own container.
-		if err := resourcecontainer.RunInResourceContainer(s.ResourceContainer); err != nil {
-			glog.Warningf("Failed to start in resource-only container %q: %v", s.ResourceContainer, err)
-		} else {
-			glog.V(2).Infof("Running in resource-only container %q", s.ResourceContainer)
-		}
+		// // Run in its own container.
+		// if err := resourcecontainer.RunInResourceContainer(s.ResourceContainer); err != nil {
+		// 	glog.Warningf("Failed to start in resource-only container %q: %v", s.ResourceContainer, err)
```

Comment:
could we drop those?
the commit message should be able to satisfy an explanation

Reply:
Done!